ffi/custom_passes.cpp

```diff
@@ -719,13 +729,22 @@ struct RefPrunePass : public FunctionPass {
         CallInst *incref,
         BasicBlock *cur_node,
         SmallVectorImpl<BasicBlock*> &path_stack,
+        unsigned &subgraph_size,
+        SmallBBSet &bad_blocks,
```

Comment:
Add parameter description to docstring.

Reply:
fixed in https://github.com/numba/llvmlite/pull/644/commits/fa6092b8de364fe7a04c00a08523cc1c1e027783